src/teensy_sketch/teensy_sketch.ino

```diff
@@ -176,3 +243,9 @@ String getIMULogString(float * data) {
 String getGyroLogString(int16_t * data) {
   return (String(data[0], DEC) + "," + String(data[1], DEC) + "," + String(data[2], DEC) + ",");
 }
+
+// Function for returning formatted GPS string
+String getGPSLogString() {
+  return (String(gps.location.lat(),6) + "," + String(gps.location.lng(),6) + ",");
```

Comment:
Yep, decimal place precision. Will make it a constant though as we may use different number rounding elsewhere